src/styles/index.scss

```diff
@@ -1,3 +1,216 @@
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
The font-family is defined as 'Roboto, sans-serif' here, but you have also defined a variable $font-family with the same value. Consider using the variable for consistency and easier maintenance.